ui/.env

```diff
@@ -6,4 +6,5 @@ SINGLE_GROUP_MODE=false
 BUCKET_NAME_SINGULAR="bucket"
 BUCKET_NAME_PLURAL="buckets"
 PLATFORM_NAME="Cobudget"
+PLATFORM_LOGO="https://pbs.twimg.com/profile_images/1019572294987976704/j_Ngi0LK_400x400.jpg"
```

Comment:
better to host this on our server instead

Reply:
How to do this?

Reply:
@alinauroz - if you add it to ui/public it will be available at https://cobudget.com/xxxx.jpg